dockerfile

```diff
@@ -0,0 +1,41 @@
+
+# Use the official Python image as a base
+FROM ghcr.io/astral-sh/uv:python3.11-bookworm-slim
+
+# Set environment variables
+ENV PYTHONUNBUFFERED=1 \
+    CONFIG_PATH=/app/config.json \
+    UVICORN_PORT=8000 \
+    UVICORN_LOG_LEVEL=info
+
+# Create and set the working directory
+WORKDIR /app
+
+# Copy source to container
+COPY . /app
+
+# Install mcpo from source
+RUN pip install .
```

Comment:
better to run this after apt-get since apt-get would be cached all the time, and only the next layers will be re-built.